demo/README.md

```diff
@@ -0,0 +1,9 @@
+## Getting Started
+
+```bash
+pnpm i
+
+pnpm dev
+```
```

Comment:
For my clarification—the _demo_ has only been verified to work with `pnpm`, but the elements package itself should work with any package manager?

Reply:
There's like 3 install flows to think about here:

1. The demo app, which we should use `pnpm` for since we have a `pnpm-lock`.
2. When _developing_ the `elements` package, which we should use `pnpm` for since we have a `pnpm-lock`.
3. _Installing_ the `elements` package, which will be on a user's site and can use any package manager that they wish to use.